scripts-dev/lint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+set -e
```

Comment:
it's conventional to put this after the comments that say what the script does

Reply:
Whoops, good point. That was ugly.